features/support/commit_helpers.rb

```diff
@@ -67,20 +67,33 @@ def create_commits commits_array
   commits_array = [commits_array] if commits_array.is_a? Hash
   normalize_commit_data commits_array
   commits_array.each do |commit_data|
-    commit_data.reverse_merge!(default_commit_attributes)
-    create_commit commit_data
+    create_commit add_default_commit_data(commit_data)
   end
 end
 
 
-def default_commit_attributes
-  {
+# Returns a commit_data structure consisting of
+# the given commit_data structure with default values added
+# rubocop:disable MethodLength
+def add_default_commit_data commit_data
+  result = commit_data.clone.reverse_merge!(
     file_name: "default file name #{SecureRandom.urlsafe_base64}",
-    file_content: 'default file content',
     message: 'default commit message',
     location: 'local and remote',
-    branch: current_branch_name
-  }
+    branch: current_branch_name,
+    file_content: commit_data[:file_content]
+  )
+  if result[:file_content].blank?
+    result[:file_content] = default_file_content_for commit_data[:file_name]
+  end
+  result
+end
+# rubocop:enable MethodLength
```

Comment:
Please stop adding these.
If needed lets bump to allowed method length, but otherwise please find a way to meet the standard.
Here it can be done with 
```ruby
def add_default_commit_data commit_data
  file_name = "default file name #{SecureRandom.urlsafe_base64}"
  file_content = default_file_content_for(commit_data[:file_name] or file_name)
  commit_data.clone.reverse_merge(
    file_name: file_name,
    message: 'default commit message',
    location: 'local and remote',
    branch: current_branch_name,
    file_content: file_content
  )
end
```